src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,16 +1,24 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
 public class Country implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
 
-    public Country() {
-    }
-
     public Country(String name) {
         this.name = name;
     }
 
+    public Country() {
+    }
```

Comment:
For the future, it is better that the constructors go in ascending order, that is, the constructor without parameters should be the first

Reply:
Ok